src/main/java/Lottery.java

```diff
@@ -0,0 +1,9 @@
+import java.util.Random;
+public class Lottery {
+    private static final Random RANDOM = new Random();
+    public static Ball getRandomBall() {

```

Comment:
https://mate-academy.github.io/jv-program-common-mistakes/java-core/oop/lottery#dont-use-static-methods-in-your-solution